<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceForm</class>
 <widget class="QWidget" name="ServiceForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>452</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>452</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

QLabel{
	font-size: 20px;
	font-weight: bold;
	color: solid white;
}


QPushButton{
	
	background: black;
	border: 2px solid white;
	color: white;
	font-size: 12px;
	font-weight: bold;
	border-radius: 6px;
}
QPushButton:hover{
	
	background: white;
	border: 2px solid black;
	color: black;
	border-radius: 6px;
}

#changeButton:hover
{
	icon:url(:/images/res/icons8-pencil-50 (1).png);
}

#deleteButton:hover
{
	icon: url(:/images/res/icons8-trash-50.png);
}

</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>87</y>
     <width>344</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="serviceNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>22</y>
     <width>326</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="servicePassLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>99</y>
     <width>330</width>
     <height>86</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="serviceButtons" native="true">
   <property name="geometry">
    <rect>
     <x>346</x>
     <y>9</y>
     <width>101</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 40px</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="changeButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/images/res/icons8-pencil-50.png</normaloff>:/images/res/icons8-pencil-50.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>46</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/images/res/icons8-trash-50 (1).png</normaloff>:/images/res/icons8-trash-50 (1).png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>46</width>
        <height>56</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>9</y>
     <width>441</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid white;
border-radius:4px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>20</width>
     <height>179</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="copyPassButton">
   <property name="geometry">
    <rect>
     <x>291</x>
     <y>119</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#copyPassButton
{
	background: transparent;
	border: 0px;
}

#copyPassButton:hover
{
	icon:url(:/images/res/icons8-copy-50.png);
	background: transparent;
	border: 0px;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/res/icons8-copy-50 (1).png</normaloff>:/images/res/icons8-copy-50 (1).png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>46</width>
     <height>56</height>
    </size>
   </property>
  </widget>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>label</zorder>
  <zorder>serviceNameLabel</zorder>
  <zorder>servicePassLabel</zorder>
  <zorder>serviceButtons</zorder>
  <zorder>copyPassButton</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
